### PR TITLE
Support passing additionalGrantParameters to OidcClient.refreshTokens and OidcClient.revokeAccessToken

### LINKED ISSUE #35422: Support passing `additionalGrantParameters` to `OidcClient.refreshTokens` and `OidcClient.revokeAccessToken`
### Description

I am using Keycloak with a client that is configured to use signed JWT for authentication. This works fine with `OidcClient.refreshTokens.getTokens()`:

```
String jwt = generateJwt();
var tokens = oidcClient.getTokens(Map.of("client_assertion", jwt, "client_assertion_type", "urn:ietf:params:oauth:client-assertion-type:jwt-bearer"));
```

However, there is no way to specify additional params when calling  `OidcClient.refreshTokens` or `OidcClient.revokeAccessToken`.

> 

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by Eng-Fouad**
Resolves #35422

### CODE REVIEW

File: extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/OidcClient.java [hunk 1]
```diff
@@ -33,14 +33,38 @@ default Uni<Tokens> getTokens() {
...
+     */
+    default Uni<Boolean> revokeAccessToken(String accessToken) {
+        return revokeAccessToken(accessToken, Collections.emptyMap());
+    }
+
+    /**
+     * Revoke the access token.
+     *
+     * @param accessToken access token which needs to be revoked
+     * @param additionalGrantParameters additional grant parameters
```
Comment (sberyozkin) [inline extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/OidcClient.java:65]: @Eng-Fouad Can you please change the name of the parameter to `additionalParameters` as revoking  a token is not a grant request.

**Comment by sberyozkin** [comment]
Thanks @Eng-Fouad Can you please tweak the `oidc-wiremock` integration test a bit to pass an extra refreshToken parameter ? This one, https://github.com/quarkusio/quarkus/blob/main/integration-tests/oidc-client-wiremock/src/main/java/io/quarkus/it/keycloak/FrontendResource.java#L66  and update wiremock expectation here, https://github.com/quarkusio/quarkus/blob/main/integration-tests/oidc-client-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakRealmResourceManager.java#L63 ?

**Comment by Eng-Fouad** [comment]
Updated code based on your comments. Please review it. Thanks.

**Review by sberyozkin — APPROVED** [review]
Thanks @Eng-Fouad

**Pull request merged**